### DISCUSSION TITLE
大佬这次升级实在太激进了，docker镜像直接挂掉了

### DISCUSSION THREAD

**Discussion by gouku** [AlistGo/alist#1686] [General]
根据之前的文档 alist:latest是稳定版，alist:main是最新版，我一直用的latest，今天docker 升级顺手把镜像升级了一下，网站直接挂掉了，一看直接升级到了v3，和v2不兼容，而且github这边v3还是rc并没正式发布，但镜像却直接pull成v3的了。然后我强制选了个v2的镜像进行降级，发现东西依然没了，于是找到之前导出的配置，但发现v3好像又不支持导入配置，崩溃了……

**Comment by bebestmaple** [comment]
同样，只能先强制使用v2版的用着先了。
